components/incidents/createIncidents/InputStatus.tsx

```diff
@@ -0,0 +1,152 @@
+import { FormControl } from "baseui/form-control";
+import styles from "../../../styles/CreateIncident.module.css";
+import { ProgressBar, SIZE } from "baseui/progress-bar";
+import { FlexGrid, FlexGridItem } from "baseui/flex-grid";
+import { Block, BlockProps } from "baseui/block";
+import React from "react";
+import { SpecialEvent, InputStatusprops } from "../../../variableTypes";
+import { STATUSNames } from "./../../../constants";
+
+function calculateStatus(status: String) {
+  if (status == "Investigating") {
+    return 0;
+  }
+  if (status == "Identified") {
+    return 33;
+  }
+  if (status == "Monitoring") {
+    return 66;
+  } else {
+    return 100;
+  }
+}
+
+export default function InputStatus(props: InputStatusprops) {
+  const itemProps: BlockProps = {
+    height: "scale1000",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    onClick: (e: Event) => {
+      const target = e.target as Element;
+      props.updateStatus(target.innerHTML!);
+    },
+    overrides: {
+      Block: {
+        style: {
+          cursor: "pointer",
+        },
+      },
+    },
+  };
+  const selectedItemProps: BlockProps = {
+    height: "scale1000",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    color: "blue",
```

Comment:
can decide this based on selected status

Reply:
Done. Now only one itemprops based on selected status